activerecord/test/cases/tasks/postgresql_rake_test.rb

```diff
@@ -205,13 +274,16 @@ def setup
           "adapter"  => "postgresql",
           "database" => "my-app-db"
         }
-
-        ActiveRecord::Base.stubs(:connection).returns(@connection)
       end
 
       def test_db_retrieves_collation
-        @connection.expects(:collation)
-        ActiveRecord::Tasks::DatabaseTasks.collation @configuration
+        ActiveRecord::Base.stub(:connection, @connection) do
+          ActiveRecord::Base.stub(:clear_active_connections!, true) do
```

Comment:
why do we need to stub `clear_active_connections!`?

Reply:
We don't! Removing.